static/styles/_components/_entity.scss

```diff
@@ -75,6 +86,47 @@
   }
 }
 
+.time-period {
+  width: 100%;
+  float: left;
+  padding: 1rem 0;
+  label {
+    width: 50%;
+    margin-right: 1rem;
+    display: inline-block;
+  }
+
+  select,
+  .chosen-container {
+    width: 50% !important;
```

Comment:
Ugh I know :) It's kind of a temporary hack because of the way the ChosenJS is working. I was having a really hard time getting the size right without it.